src/applications/personalization/profile/components/direct-deposit/DirectDepositV2.jsx

```diff
@@ -14,31 +23,46 @@ import PaymentHistory from './PaymentHistory';
 import BankInfoCNPv2 from './BankInfoCNPv2';
 import BankInfoEDU from './BankInfoEDU';
 
-const DirectDeposit = () => {
-  // const [showSaveSucceededAlert, setShowSaveSucceededAlert] = React.useState(
-  //   false,
-  // );
+const DirectDeposit = ({ cnpUiState, eduUiState }) => {
+  const [recentlySavedBankInfo, setRecentlySavedBankInfo] = React.useState('');
 
-  // const isSavingBankInfo = directDepositUiState.isSaving;
-  // const wasSavingBankInfo = usePrevious(directDepositUiState.isSaving);
+  const isSavingCNPBankInfo = cnpUiState.isSaving;
+  const wasSavingCNPBankInfo = usePrevious(cnpUiState.isSaving);
+  const cnpSaveError = cnpUiState.responseError;
+  const isSavingEDUBankInfo = eduUiState.isSaving;
+  const wasSavingEDUBankInfo = usePrevious(eduUiState.isSaving);
+  const eduSaveError = eduUiState.responseError;
 
   React.useEffect(() => {
     focusElement('[data-focus-target]');
     document.title = `Direct Deposit | Veterans Affairs`;
   }, []);
 
-  // show the user a success alert after their bank info has saved
-  // React.useEffect(
-  //   () => {
-  //     if (wasSavingBankInfo && !isSavingBankInfo && !saveError) {
-  //       setShowSaveSucceededAlert(true);
-  //       setTimeout(() => {
-  //         setShowSaveSucceededAlert(false);
-  //       }, 6000);
-  //     }
-  //   },
-  //   [wasSavingBankInfo, isSavingBankInfo, saveError],
-  // );
+  // show the user a success alert after their CNP bank info has saved
+  React.useEffect(
+    () => {
+      if (wasSavingCNPBankInfo && !isSavingCNPBankInfo && !cnpSaveError) {
+        setRecentlySavedBankInfo('compensation and pension benefits');
+        setTimeout(() => {
```

Comment:
What do you think about putting
```
setTimeout(() => {
   setRecentlySavedBankInfo('');
}, 6000);
```
into a method that's called something like clearSaveSuccessAlert?

Reply:
sure